app/controllers/actions/JourneyAction.scala

```diff
@@ -30,10 +30,9 @@ import uk.gov.hmrc.play.HeaderCarrierConverter
 
 import scala.concurrent.{ExecutionContext, Future}
 
-case class JourneyAction @Inject()(cacheService: ExportsCacheService, mcc: MessagesControllerComponents)
-    extends ActionRefiner[AuthenticatedRequest, JourneyRequest] with SessionIdAware {
-
-  implicit override val executionContext: ExecutionContext = mcc.executionContext
+case class JourneyAction @Inject()(cacheService: ExportsCacheService)(
```

Comment:
Unrelated to this change but this probably doesn't need to be a `case class` as we dont need any of the benefits that brings